### PR TITLE
added req42 as template

### PR DESCRIPTION
**Pull request by rdmueller**
not perfect, but works

### CODE REVIEW

File: scripts/downloadTemplate.gradle [hunk 1]
```diff
@@ -30,15 +37,15 @@ task downloadTemplate(
             template = templates.keySet()[(ant.template as Integer)-1]
         }
         println "${color 'green', 'Install '+template+' documentation template.'}"
-        def targetDir = new File(new File(docDir, inputPath), 'arc42')
+        def targetDir = new File(new File(docDir, inputPath), template)
         targetDir.mkdirs()
-        if (template=='arc42') {
-            println "For more information about arc42 see https://arc42.org"
-            /**
-             if (targetDir.exists()) {println "${targetDir.canonicalPath} already exists"
```
Comment (PacoVK) [inline scripts/downloadTemplate.gradle:38]: you removed this, i think it was useful to let the user know that there is already a setup and the intentional re-install requires manually removal of the existing structure

File: scripts/downloadTemplate.gradle [hunk 2]
```diff
@@ -30,15 +37,15 @@ task downloadTemplate(
...
-             println "in order to re-install the template, please remove the folder first and re-run the script"
-             throw new RuntimeException("Template folder already exists")}**/
-            ant.input(message: "${color 'green', 'Which language do you want to install?'}", validargs: 'CZ,EN,DE,ES,IT,NL,UA', addproperty: 'lang')
+        if (template in ['arc42', 'req42']) {
```
Comment (PacoVK) [inline scripts/downloadTemplate.gradle:42]: isn't that already ensured with line 36-37?

File: scripts/downloadTemplate.gradle (hunk 1 above)
Comment (rdmueller) [reply scripts/downloadTemplate.gradle:38]: It was commented out. Yes, the origianl intention was to warn the user. But as averathing is under source control, I originally commented it out and now cleaned the code up

Comment (rdmueller) [reply scripts/downloadTemplate.gradle:38]: But if you think it is useful, we can re-enable it

File: scripts/downloadTemplate.gradle [hunk 3]
```diff
@@ -30,15 +37,15 @@ task downloadTemplate(
...
+            if (template=='arc42') {
+                println "For more information about arc42 see https://arc42.org"
+            } else {
```
Comment (PacoVK) [inline scripts/downloadTemplate.gradle:45]:
```suggestion
            } else if (template == 'req42') {
```
We could throw an exception and abort in the default `else` here, if L36-37 does not already cover that
Alternative: use a switch case, throw an exception in the `default` block - but that just a matter of personal preference i guess

File: scripts/downloadTemplate.gradle (hunk 2 above)
Comment (rdmueller) [reply scripts/downloadTemplate.gradle:42]: Line 23 allows to configure more local templates which are not part of this list. Those templates currently can't have a language or other features. Instead, you have to configure one template for each feature

**Review by rdmueller — COMMENTED** [review]
Thanx for your review!

File: scripts/downloadTemplate.gradle (hunk 1 above)
Comment (PacoVK) [reply scripts/downloadTemplate.gradle:38]: Argh, did't recognize the block comment. However, i do think it's useful to make this explicit to the user, that re-install is not possible on existing projects

File: scripts/downloadTemplate.gradle (hunk 3 above)
Comment (PacoVK) [reply scripts/downloadTemplate.gradle:45]: from your comments above,  i think we can ignore my suggestion then

**Review by PacoVK — APPROVED** [review]
LGTM

**Pull request merged**